test/Microsoft.Azure.Devices.Edge.Test.Common/EdgeRuntime.cs

```diff
@@ -44,11 +44,11 @@ public EdgeRuntime(string deviceId, Option<string> agentImage, Option<string> hu
 
             if (nestedEdge == true)
             {
-                hubEnvVar = new[] { ("RuntimeLogLevel", "debug"), ("experimentalFeatures__enabled", "true"), ("experimentalFeatures__nestedEdgeEnabled", "true"), ("DeviceScopeCacheRefreshDelaySecs", "1") };
+                hubEnvVar = new[] { ("RuntimeLogLevel", "debug"), ("NestedEdgeEnabled", "true"), ("DeviceScopeCacheRefreshDelaySecs", "1") };
```

Comment:
>("NestedEdgeEnabled", "true"), [](start = 66, length = 30)

in the spirit of the other deployment changes, we can prob just omit this altogether now